plugins/smartling/src/plugin.tsx

```diff
@@ -439,13 +439,27 @@ registerPlugin(
           return api.getProject(id).then(res => transformProject(res.project));
         },
         search(q = '') {
-          return api
-            .getAllProjects()
-            .then(res =>
-              res.results
-                .filter(proj => proj.projectName.toLowerCase().includes(q.toLowerCase()))
-                .map(transformProject)
-            );
+          return api.getAllProjects().then(res => {
+            if (!res.results || res.results.length === 0) {
+              return appState.globalState
+                .getPluginPrivateKey(pkg.name)
+                .then((pluginPrivateKey: string) => {
+                  if (api.isPluginPrivateKeySame(pluginPrivateKey)) {
+                    appState.snackBar.show('Oh no! There was an error searching for resources');
```

Comment:
can you validate with Sajal on the message? Is "oh no" on brand for us? I haven't seen that anywhere else

Reply:
I ahve just reused the message thats already there in ResourcePicker as was the case previously. Only updated the message in this particular scenario
<img width="576" alt="image" src="https://github.com/user-attachments/assets/624aa15c-77de-4c28-a863-a8a56b7d06d5">

Reply:
There are a dozen of places in webapp where "oh no! ..." message are used..
<img width="268" alt="image" src="https://github.com/user-attachments/assets/10e9ff55-39fc-473e-8832-411145a38771">